src/linter.js

```diff
@@ -19,7 +19,9 @@ export default function linter(options, compiler, callback) {
         callback();
       }
     })
-    .catch(callback);
+    .catch(() => {
+      callback();
```

Comment:
We can put a condition here to fail if `failOnError` is enabled.
```js
.catch((e) => {
  if (options.failOnError) {
    callback(e);
  } else {
    callback();
  }
});
```

Reply:
Yes,that is better